src/config/env.js

```diff
@@ -0,0 +1,3 @@
+const { GIT_PROXY_SERVER_PORT = 8000, GIT_PROXY_UI_PORT = 8080 } = process.env;
```

Comment:
I assume this shorthand means that if object destructuring results in `undefined` or `null` values, then the `const` will be assigned the specific values written?

Reply:
in general,

object detructuring default value is only set when the value is `undefined`. If a value is null in javascript, that means it was most likely intentional.

When an environment variable is not set, it will be `undefined`. Hence why this shorthand works.

Reply:
Makes sense, nice and clear clarification.

Thank you @lwhiteley 👍